client/landing/gutenboarding/onboarding-block/acquire-intent/index.tsx

```diff
@@ -51,8 +50,8 @@ const AcquireIntent: React.FunctionComponent = () => {
 		handleSiteTitleSubmit();
 	};
 
-	// translators: Button label for advancing to Design Picker step in onboarding
-	const nextLabel = __( 'Choose design' );
+	// translators: Button label for advancing to second step in onboarding
```

Comment:
```suggestion
	// translators: Button label for advancing to the second step in onboarding
```

Reply:
made it generic now as part of [ActionButtons](https://github.com/Automattic/wp-calypso/pull/43136/files#diff-6ae3fac2fb17d29ef4858d096d249a13R58)